lib/src/views/main_view.dart

```diff
@@ -86,10 +87,12 @@ class _MainViewTabData {
   Widget toView() => view;
 
   Tab toTab(BuildContext context, bool isActive) => Tab(
-        icon: SvgPicture.asset(
-          svgAsset,
-          color: isActive ? Theme.of(context).tabBarTheme.labelColor : Theme.of(context).tabBarTheme.unselectedLabelColor,
-          semanticsLabel: label(context) as String,
+        icon: SimpleTooltip(
+          message: label(context) as String,
+          child: SvgPicture.asset(
+            svgAsset,
+            color: isActive ? Theme.of(context).tabBarTheme.labelColor : Theme.of(context).tabBarTheme.unselectedLabelColor,
```

Comment:
Just noticed there is still a `Theme.of()` here but I think it's acceptable in this case, as we did specify a `TabBarTheme` to be reused for every "text tab bar" and this one using icons is just an exception, but should follow the same rules.